doc.txt

```diff
@@ -1,3 +1,20 @@
+/**
+ * @defgroup configurations Compile time configurations
```

Comment:
Why not call the group `config`. Would save us some time when typing and also some head-ache regarding line-length ;-).